OpenRA.Mods.Common/Traits/ExternalCaptures.cs

```diff
@@ -128,14 +128,16 @@ public override bool CanTargetActor(Actor self, Actor target, TargetModifiers mo
 		{
 			var c = target.TraitOrDefault<ExternalCapturable>();
 
-			var canTargetActor = c != null && !c.CaptureInProgress && c.Info.CanBeTargetedBy(self, target.Owner);
+			var canTargetActor = c != null && !c.CaptureInProgress && c.CanBeTargetedBy(self, target.Owner);
 			var capturesInfo = self.Trait<ExternalCaptures>().Info;
 			cursor = canTargetActor ? capturesInfo.CaptureCursor : capturesInfo.CaptureBlockedCursor;
 			return canTargetActor;
 		}
 
 		public override bool CanTargetFrozenActor(Actor self, FrozenActor target, TargetModifiers modifiers, ref string cursor)
 		{
+			// Using ExternalCapturableInfo.CanBeTargetedBy skips check for trait enabledness of the frozen actor.
```

Comment:
The problem is that this doesn't actually work.  If the trait is disabled then the order will go through but then be immediately ignored on the resolve side.  Like I mentioned earlier, I can live with this behaviour for the sake of actors that don't freeze under fog, but I will push a modification to the PR that replaces these comments with something more appropriate.